doc/INSTALL.md

```diff
@@ -235,6 +236,34 @@ Build with:
       make PIE=1 DEVELOPER=0 \
       CONFIGURATOR_CC="arm-linux-androideabi-clang -static"
 
+To cross-compile for Raspberry Pi
+--------------------
+
+Obtain the [official Raspberry Pi toolchains](https://github.com/raspberrypi/tools). This document assumes compilation will occur towards the Raspberry Pi 3 (arm-linux-gnueabihf as of Mar. 2018). In addition, obtain and install cross-compiled versions of sqlite 3 and gmp.
+
+Depending on your toolchain location and target arch, source env variables will need to be set. They can be set from the command line as such:
+
+    export PATH=$PATH:/path/to/arm-linux-gnueabihf/bin
+    # Change next line depending on specific Raspberry Pi device
+    target_host=arm-linux-gnueabihf
```

Comment:
if should be export target_host=arm-linux-gnueabihf?

Reply:
What I wrote works on Ubuntu 17.10. I can't comment on other OSes.